src/Pusher.php

```diff
@@ -745,4 +751,46 @@ public function notify($interests, $data = array(), $debug = false)
 
         return false;
     }
+
+    /**
+     * Verify that a webhook actually came from Pusher, and marshals them into a Webhook object.
+     *
+     * @param array  $headers an array of headers from the request (for example, from getallheaders())
+     * @param string $body    the body of the request (for example, from file_get_contents('php://input'))
+     *
+     * @return Webhook object with the properties time_ms (an int) and events (an array of event objects)
+     */
+    public function webhook($headers, $body)
+    {
+        $this->ensure_valid_signature($headers, $body);
+        $decoded_events = array();
```

Comment:
This appears unused.

Reply:
As in the webhook function? That's what the user will call in order to validate the signature and parse the webhooks - as in the Go lib.

Reply:
Oh you mean $decoded_events and are totally right, derp!